instance/github.py

```diff
@@ -157,8 +164,39 @@ def __init__(self, number, fork_name, branch_name, title, username, body=''):
         self.body = body
 
     @property
+    def settings(self):
+        """
+        Settings contained in the PR body
+        """
+        return yaml.load(get_settings_from_pr_body(self.body)) or {}
+
+    @property
+    def database_url(self):
```

Comment:
Could we improve the consistency of the configuration variable names? mongodb is a database too. Maybe `RELATIONAL_DATABASE_URL` and `DOCUMENT_DATABASE_URL`? Or simply `MYSQL_URL` and `MONGO_URL`.